src/sphinxtheme/plone/plone_basic/layout.html

```diff
@@ -102,6 +102,8 @@
   <link rel="icon" href="{{ pathto('_static/' + theme_favicon, 1) }}" type="image/x-icon">
   <link rel="shortcut icon" type="image/x-icon" href="{{ pathto('_static/' + theme_favicon, 1) }}" />
   <link type="image/x-icon" href="{{ pathto('_static/' + theme_favicon, 1) }}" />
+  <link rel="stylesheet" type="text/css" href="static/css/ploneorg5.css">
```

Comment:
Revert this. It is included through a configuration file at https://github.com/plone/sphinxtheme.plone/blob/a0ea9d4a1b7f87f5bb26efa828b70fd15f6d4f30/src/sphinxtheme/plone/plone_org_5/theme.conf.